lib/mix/tasks/coh.clean.ex

```diff
@@ -304,6 +304,10 @@ defmodule Mix.Tasks.Coh.Clean do
 
   defp remove!(config, _), do: config
 
+  defp migrations_path(repo) do
+    "priv/repo/migrations/"
+  end
```

Comment:
There were errors running `mix test` re: `migrations_path/1` being undefined. So, I added these embarrassing hacks to see if the tests would pass :)

The tests do pass now but it seems like there should be a better solution for `migrations_path/1` being undefined.